ui-core/src/main/java/com/adyen/checkout/components/extensions/ViewExtensions.kt

```diff
@@ -36,13 +36,10 @@ fun TextView.setLocalizedTextFromStyle(
     val attrs = intArrayOf(android.R.attr.text)
     val typedArray = localizedContext.obtainStyledAttributes(styleResId, attrs)
     val stringResValue = typedArray.getString(0)
-    text = if (formatHyperLink) {
-        // check if the string contains the replacement token twice
-        val counter = stringResValue?.split(replacementToken)?.size?.minus(1)
-        if (counter != 2) return
-        val spannable = stringResValue.formatStringWithHyperlink(replacementToken)
-        spannable
-    } else stringResValue
+    // check if the string contains the replacement token twice
+    val counter = stringResValue?.split(replacementToken)?.size?.minus(1)
```

Comment:
Should we move this whole logic to inside `formatStringWithHyperlink`? `setLocalizedTextFromStyle` shouldn't really know about the logic of hyperlink formatting, including counting how many times did the token appear. What do you think?